charm/kafka/reactive/tls.py

```diff
@@ -163,3 +163,63 @@ def import_ca_crt_to_keystore():
 
             remove_state('tls_client.ca_installed')
             set_state('kafka.ca.keystore.saved')
+
+
+@when('endpoint.certificates.departed')
+def clear_certificates():
+    ca_path = '/usr/local/share/ca-certificates/{}.crt'.format(
+        hookenv.service_name()
+    )
```

Comment:
Might refactor this to a common function or staticmethod that resolves the ca_path for other handlers (such as import_ca_cert_to_keystore above).

Reply:
done